### PR TITLE
Feat / ADF-436 / [GREP] - Usage - Update properties UI on selected resource

### PR DESCRIPTION
**Pull request by 404passionFruit**
Jira https://oat-sa.atlassian.net/browse/ADF-436

In order to test:

- Create a class and go to Manage schema
- Add two properties (one will be primary and one will be secondary)
- Set Type to list - Single/Multiple choice - Search/Dropdown
- Set List values of primary to Parent list
- Set List values of secondary to child list
- Save changes and reload Manage Schema
- Set Depends on property of Secondary property to Primary
- Save changes and create item

Accepted criteria:

- You should be able to ONLY select secondary properties when a primary property has value. If not, secondary input should be disabled.
- If you have assigned values to both properties and then set primary to empty, secondary should be cleared and disabled.
- All changes should be persisted

If you want to directly test AC's without setting up the classes and properties you can access to class ADF-436 and item ADF-436 1 directly (which already has dependent properties defined)

TODO: _(Only for future reference purposes, can be ignored for PR review)_
- [x] Set proper primary URI on BE https://github.com/oat-sa/tao-core/pull/3103/files#diff-d663f726bfddec08e0b9a1cc9181ba9ecc8416358b2a1b2f0ad2f2721e063977R175
- [x] Change https://github.com/oat-sa/tao-core/pull/3103/files#diff-d663f726bfddec08e0b9a1cc9181ba9ecc8416358b2a1b2f0ad2f2721e063977R176 when BE is ready
- [x] Test secondary dropdown single choice reactivity (should clear when primary is cleared), fix if not working properly
- [x] (Optional) Refactor
- [ ] (Optional) Move styles and js to tao-core-ui-fe

### CODE REVIEW

File: views/js/form/post-render-props.js [hunk 1]
```diff
@@ -308,6 +309,9 @@ define([
             $properties = $container.children('div[id*="property_"]').not('.property-block');
         }
         if (!$properties.length) {
+            if ($container.children('[name="tao.forms.instance"]').length) {
+                secondaryProps.move($container)
```
Comment (habemuscode) [inline views/js/form/post-render-props.js:313]: Missing semicolon

File: views/js/form/secondary-property.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                    $(this).find('option[value=" "]').attr('selected', 'selected');
+                    $(this).trigger('change');
+                });
+                return;
+            }
+
+            const $inputElt = $(this).find('input');
+            if ($inputElt.length) {
+                $inputElt.each(function() {
+                    $(this).val(null).trigger('change')
```
Comment (habemuscode) [inline views/js/form/secondary-property.js:71]: Missing semicolon

File: views/js/form/secondary-property.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    function moveSecondaryProperty($prop, $props) {
+        const primaryPropUri = $prop.find('[data-depends-on-property]').data('depends-on-property');
+        let $primaryProp = $($props.filter(function() {
+            return !!$(this).find(`#${primaryPropUri}`).length;
+        })[0]);
+
+        if(!$primaryProp.length) {
+            console.error('Primary property not found', primaryPropUri)
```
Comment (habemuscode) [inline views/js/form/secondary-property.js:84]: Console output?

And missing semicolon

**Review by habemuscode — APPROVED** [review]
LGTM, just a few comments

**Review by Silvia-loza — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on playground (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**